src/DIPS.Xamarin.UI/Controls/TrendGraph/TrendGraph.xaml.cs

```diff
@@ -110,35 +117,21 @@ private static void OnAnyPropertyChanged(BindableObject bindable, object oldvalu
             trendGraph.StartRedraw(trendGraph, EventArgs.Empty);
         }
 
-        /// <summary>
-        ///  <see cref="GraphColor" />
-        /// </summary>
-        public static readonly BindableProperty GraphColorProperty =
-            BindableProperty.Create(nameof(GraphColor), typeof(Color), typeof(TrendGraph), Color.FromHex("#a26eba"), propertyChanged: OnAnyPropertyChanged);
-
         /// <summary>
         /// Color of each graph
         /// </summary>
-        public Color GraphColor
-        {
-            get { return (Color)GetValue(GraphColorProperty); }
-            set { SetValue(GraphColorProperty, value); }
-        }
+        public Color GraphColor { get; set; } = Color.FromHex("#a26eba");
 
         /// <summary>
-        ///  <see cref="GraphBackgroundColor" />
+        /// Background color of each graph
         /// </summary>
-        public static readonly BindableProperty GraphBackgroundColorProperty =
-            BindableProperty.Create(nameof(GraphBackgroundColor), typeof(Color), typeof(TrendGraph), Color.FromHex("#edf3f4"), propertyChanged: OnAnyPropertyChanged);
+        public Color GraphBackgroundColor { get; set; } = Color.FromHex("#edf3f4");
 
         /// <summary>
-        /// Background color of each graph
+        /// Color to draw the graph with if the value is outside the reference area.
         /// </summary>
-        public Color GraphBackgroundColor
-        {
-            get { return (Color)GetValue(GraphBackgroundColorProperty); }
-            set { SetValue(GraphBackgroundColorProperty, value); }
-        }
+        public Color ReferenceColor { get; set; } = Color.Red;
```

Comment:
I propose calling this `OutOfBoundsColor `  because what determines the color is called `LowerBound` and `UpperBound`.